modules/covid_research/app/controllers/covid_research/volunteer/submissions_controller.rb

```diff
@@ -26,6 +27,28 @@ def create
         end
       end
 
+      def update
+        form_service = FormService.new('COVID-VACCINE-TRIAL-UPDATE')
+        puts 'Payload in update: '
+        puts payload
+        with_monitoring do
+          if form_service.valid?(payload)
+            ConfirmationMailerJob.perform_async(payload['email'])
```

Comment:
is this specifying a certain email job for an update submission vs. an initial submission?

Reply:
@batemapf not yet.  There is a lot more functionality to add to support `update`.  Scope of this PR is the initial endpoint to receive and validate an `update` form.  Follow on PRs will cover the mailer job, data storage and sending to Genesis.